cypress/integration/alerts_dashboard_flyout_spec.js

```diff
@@ -0,0 +1,287 @@
+        'Trigger name'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_triggerName_${QUERY_TRIGGER}"]`).contains(
+        QUERY_TRIGGER
+      );
+
+      // Confirm 'Severity' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_severity_${QUERY_TRIGGER}"]`).contains(
+        'Severity'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_severity_${QUERY_TRIGGER}"]`).contains(
+        '2 (High)'
+      );
+
+      // Confirm 'Monitor' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_monitor_${QUERY_TRIGGER}"]`).contains(
+        'Monitor'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_monitor_${QUERY_TRIGGER}"]`).contains(
+        QUERY_MONITOR
+      );
+
+      // Confirm 'Conditions' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_conditions_${QUERY_TRIGGER}"]`).contains(
+        'Condition'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_conditions_${QUERY_TRIGGER}"]`).contains(
+        `ctx.results[0].hits.total.value < 10000`
+      );
+
+      // Confirm 'Time range for the last' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_timeRange_${QUERY_TRIGGER}"]`).contains(
+        'Time range for the last'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_timeRange_${QUERY_TRIGGER}"]`).contains(
+        '10 day(s)'
+      );
+
+      // Confirm 'Filters' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_filters_${QUERY_TRIGGER}"]`).contains(
+        'Filters'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_filters_${QUERY_TRIGGER}"]`).contains('-');
+
+      // Confirm 'Group by' sections renders as expected.
+      cy.get(`[data-test-subj="alertsDashboardFlyout_groupBy_${QUERY_TRIGGER}"]`).contains(
+        'Group by'
+      );
+      cy.get(`[data-test-subj="alertsDashboardFlyout_groupBy_${QUERY_TRIGGER}"]`).contains('user');
+
+      // Set the 'severity' filter to only display ACTIVE alerts.
+      cy.get('[data-test-subj="dashboardAlertStateFilter"]').select('Active');
+      cy.wait(10000);
+
+      // Confirm the flyout dashboard contains 1 alert.
+      cy.get('tbody > tr').should(($tr) => expect($tr).to.have.length(1));
+
+      // Select the alert.
+      cy.get('input[data-test-subj^="checkboxSelectRow-"]').first().click();
+
+      // Press the flyout 'Acknowledge button, and wait for the AcknowledgeAlerts API call to complete.
+      cy.get('[data-test-subj="flyoutAcknowledgeAlertsButton"]').click();
+    });
+
+    // Confirm acknowledge alerts toast displays expected text.
+    cy.contains('Successfully acknowledged 1 alert.');
+
+    // Confirm alerts were acknowledged as expected.
+    cy.get(`[data-test-subj="alertsDashboardFlyout_${QUERY_TRIGGER}"]`).within(() => {
+      // Wait for GetAlerts API call to complete.
+      cy.wait(10000);
```

Comment:
This can also be removed if next step is setting filter

Reply:
Removed this `wait` period.